content/docs/command-reference/exp/run.md

```diff
@@ -143,6 +144,10 @@ committing them to the Git repo. Unnecessary ones can be [cleared] with
 - `-C <path>`, `--copy-paths <path>` - List of ignored or untracked paths to
   copy into the temp directory. Only used if `--temp` or `--queue` is specified.
 
+- `-M <message>`, `--message <message>` - Custom commit message to use when
```

Comment:
is `-m` taken? (git has `-m`, so can be easier a bit)

Reply:
should we consider overriding this for `dvc exp run`? And get rid of it in 3.0? I don't even remember why did we have it. I think it was way before exp show, VS Code, Studio, etc.

Reply:
Will do it in 3.0 (which should be soon) https://github.com/iterative/dvc/pull/9458

Reply:
does it means we'll have to rename it again then? (feels it's better to disable it on `dvc exp run` now tbh)

Reply:
Why not merge with only `--message` and add `-m` when ready to break? That way, you don't need to wait for 3.0 for this feature.

Reply:
Updated without the short flag for now.